packages/hoppscotch-sh-admin/src/components/settings/ServerRestart.vue

```diff
@@ -72,13 +72,14 @@ onMounted(async () => {
     success = await resetInfraConfigs(resetInfraConfigsMutation);
     if (!success) return;
   } else {
-    const authResult = await updateAuthProvider(
-      updateAllowedAuthProviderMutation
-    );
     const infraResult = await updateInfraConfigs(updateInfraConfigsMutation);
 
-    success = authResult && infraResult;
-    if (!success) return;
+    if (infraResult) {
+      const authResult = await updateAuthProvider(
+        updateAllowedAuthProviderMutation
+      );
+      if (!authResult) return;
+    } else return;
```

Comment:
```suggestion
    if (!infraResult) {
      return;
    }

    const authResult = await updateAuthProvider(
      updateAllowedAuthProviderMutation
    );
    if (!authResult) {
      return
    }
```

We could perform an early return instead of the trailing `else` block.